src/components/Navigation.jsx

```diff
@@ -28,14 +64,39 @@ function Navigation() {
 
       {/* Dropdown menu */}
       {menuOpen && (
-        <div className="absolute top-0 right-0 bg-white shadow-lg pr-9 pt-9 z-40 md:hidden">
-          <ul className="flex flex-col p-4">
+        <div className="absolute top-0 right-0 bg-white shadow-lg pr-9 pt-12 pl-5 pb-5 z-40 md:hidden p-3">
+          {userData && (
+            <div className="py-3 items-center mb-4 border-b">
+              <div>
+                <Link to="/account" className="flex space-x-2 border-gray-500">
+                  <img
```

Comment:
If this image is being used below as well, let's pull out into a separate component (can be in this same file) maybe called `ProfileImage` so you aren't repeating code

Reply:
Done!